parachain/pallets/inbound-queue/src/test.rs

```diff
@@ -89,6 +91,35 @@ impl pallet_balances::Config for Test {
 	type MaxHolds = ();
 }
 
+parameter_types! {
+	pub const ExecutionHeadersPruneThreshold: u32 = 10;
+	pub const ChainForkVersions: ForkVersions = ForkVersions{
+		genesis: Fork {
+			version: [0, 0, 0, 1], // 0x00000001
+			epoch: 0,
+		},
+		altair: Fork {
+			version: [1, 0, 0, 1], // 0x01000001
+			epoch: 0,
+		},
+		bellatrix: Fork {
+			version: [2, 0, 0, 1], // 0x02000001
+			epoch: 0,
+		},
+		capella: Fork {
+			version: [3, 0, 0, 1], // 0x03000001
+			epoch: 0,
+		},
+	};
+}
+
+impl snowbridge_ethereum_beacon_client::Config for Test {
+	type RuntimeEvent = RuntimeEvent;
+	type ForkVersions = ChainForkVersions;
+	type MaxExecutionHeadersToKeep = ExecutionHeadersPruneThreshold;
+	type WeightInfo = ();
+}
+
 // Mock verifier
 pub struct MockVerifier;
 
```

Comment:
Would prefer not use `mockVerifier` here because it's the core logic and maybe the most expensive call in `submit` and should be benchmarked as true as possible.

Reply:
`MockVerifier` is only used for running the benchmark tests as tests, not running the actual benchmark. The actual benchmark executes the `verify` method in `impls`.

Reply:
Corret. But since you've already linked  `EthereumBeaconClient` as direct dependendy and initialized `ExecutionHeaderBuffer` storage so IMO `MockVerifier` is not required anymore. 

I would assume just config `type Verifier = snowbridge_ethereum_beacon_client::Pallet<Test>;` in test runtime should be fine. It will also increase the test coverage seems we don't have tests for verify logic yet.